packages/amplify-e2e-core/src/categories/lambda-layer.ts

```diff
@@ -32,7 +32,67 @@ export function addLayer(cwd: string, settings?: any) {
     chain.wait('Select up to 5 compatible runtimes:');
     multiSelect(chain, runtimeDisplayNames, runtimeChoices);
     chain.wait('Who should have permission to use this layer?');
-    singleSelect(chain, settings.permission, permissionChoices);
+    multiSelect(chain, settings.permission, permissionChoices);
+
+    const layerDirRegex = new RegExp('.*/amplify/backend/function/' + settings.layerName);
+
+    chain
+      .wait('Lambda layer folders & files created:')
+      .wait(layerDirRegex)
+      .wait('Next steps:')
+      .wait('Move your libraries in the following folder:');
+
+    for (let i = 0; i < settings.runtimes.length; ++i) {
+      let layerRuntimeDirRegex = new RegExp(
+        `\\[${runtimeDisplayNames[i]}\\]: ` +
+          '.*/amplify/backend/function/' +
+          settings.layerName +
+          '/(?:src|bin)/' +
+          settings.runtimes[i] +
+          '/*',
+      );
+      chain.wait(layerRuntimeDirRegex);
+    }
+
+    chain
+      .wait('Include any files you want to share across runtimes in this folder:')
+      .wait('"amplify function update <function-name>" - configure a function with this Lambda layer')
+      .wait('"amplify push" builds all of your local backend resources and provisions them in the cloud')
+      .sendEof()
```

Comment:
Ideally we would refactor this duplicated test code into its own function

Reply:
done